bindings/nodejs/src/functions/i2c.cc

```diff
@@ -297,6 +304,7 @@ NAN_METHOD(bind_sol_i2c_write_quick)
     if (!i2c_pending) {
         delete callback;
         hijack_unref();
+        info.GetReturnValue().Set(ReverseLookupConstant("E", abs(errno)));
```

Comment:
I will fix this in the native side by setting a positive value to `errno`. @cabelitos hope it is okay with you.